packages/web/src/services/audius-sdk/auth.ts

```diff
@@ -2,20 +2,53 @@ import {
   createAuthService,
   createHedgehogSolanaWalletService
 } from '@audius/common/services'
-import { createHedgehogWalletClient } from '@audius/sdk'
+import {
+  audiusWalletActions,
+  createHedgehogWalletClient,
+  type AudiusAccount,
+  type AudiusWalletClient
+} from '@audius/sdk'
+import { metaMask } from '@wagmi/connectors'
+import { connect, createConfig, getWalletClient } from '@wagmi/core'
+import { createClient, custom, type Chain, type CustomTransport } from 'viem'
 
 import { env } from '../env'
 import { localStorage } from '../local-storage'
 
+const audiusChain = {
+  id: 1056801,
+  name: 'Audius',
+  nativeCurrency: { name: '-', symbol: '-', decimals: 0 },
+  rpcUrls: {
+    default: { http: ['https://discoveryprovider.staging.audius.co/chain'] }
+  }
+} as const satisfies Chain
+
+export const getAudiusWalletClient = async (): Promise<AudiusWalletClient> => {
+  const useMetaMask = await localStorage.getItem('useMetaMask')
```

Comment:
It's still weird to me on web that our localStorage adapter is async.... 😁 I understand that it's because the mobile version has to be and we want a common interface.